ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Models/ApplicationForm/Sections/FinanceSectionTests.cs

```diff
@@ -1,68 +1,147 @@
 using ApplyToBecome.Data.Models.Application;
+using ApplyToBecomeInternal.Extensions;
 using ApplyToBecomeInternal.Models.ApplicationForm;
 using ApplyToBecomeInternal.Models.ApplicationForm.Sections;
+using AutoFixture;
 using FluentAssertions;
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using Xunit;
 
 namespace ApplyToBecomeInternal.Tests.Models.ApplicationForm.Sections
 {
+	public static class ExtensionMethods
+	{		
+		public static void SetDeficitSurplusStatuses(this ApplyingSchool application, string status)
```

Comment:
Can you add a comment explaining the business process behind setting all statuses to be the same?

Reply:
In the test? It's testing all the deficit fields at the same time rather than one at a time in multiple tests.